src/interface/variable_pack.cpp

```diff
@@ -0,0 +1,34 @@
+//========================================================================================
+// (C) (or copyright) 2020-2021. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//=======================================================================================
+
+#include "variable_pack.hpp"
+#include "variable.hpp"
+
+namespace parthenon {
+
+template class VariablePack<Real>;
+template class VariableFluxPack<Real>;
+template void FillVarView<Real>(const vpack_types::VarList<Real> &vars,
+                                PackIndexMap *vmap, ViewOfParArrays<Real> &cv,
+                                ParArray1D<int> &sparse_assoc,
+                                ParArray1D<int> &vector_component, bool coarse = false);
```

Comment:
sometimes I get errors when including default arguments in the implementation file. Should they be removed?

Reply:
I'm not familiar with that issue but if you've had problems lets just remove the default args (I just did this)